doc/api/esm.md

```diff
@@ -572,8 +572,14 @@ The resolve hook returns the resolved file URL and module format for a
 given module specifier and parent file URL:
 
 ```js
-const baseURL = new URL(`${process.cwd()}/`, 'file://');
-
+import { URL, pathToFileURL } from 'url';
+const baseURL = pathToFileURL(process.cwd()).href;
+
+/**
+ * @param specifier {string}
```

Comment:
Can you please swap the type and the name of the parameters to follow JSDoc syntax?

Reply:
done